src/v/syschecks/syschecks.cc

```diff
@@ -46,9 +46,9 @@ void memory(bool ignore) {
     auto line = fmt::format(
       "Memory: '{}' below recommended: '{}'", shard_mem, kMinMemory);
     checklog.error(line.c_str());
-    if (!ignore) {
-        throw std::runtime_error(line);
-    }
+    // if (!ignore) {
+    //     throw std::runtime_error(line);
+    // }
```

Comment:
In general unused code should be deleted rather than commented out

Reply:
Done. True.